README.rst

```diff
@@ -1,4 +1,51 @@
 publish - Tool to create a release of a Python package
-------------------------------------------------------
+======================================================
 
-TODO
+|Build Status| |Codacy Quality Badge| |Codacy Coverage Badge|
+
+
+.. |Build Status| image:: https://semaphoreci.com/api/v1/fiaas/publish/branches/master/badge.svg
+    :target: https://semaphoreci.com/fiaas/publish
+.. |Codacy Quality Badge| image:: https://api.codacy.com/project/badge/Grade/bd7d31c7ceac43eb81884b2adc4ba3ed
+    :target: https://www.codacy.com/app/fiaas/publish?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=fiaas/publish&amp;utm_campaign=Badge_Grade
+.. |Codacy Coverage Badge| image:: https://api.codacy.com/project/badge/Coverage/bd7d31c7ceac43eb81884b2adc4ba3ed
+    :target: https://www.codacy.com/app/fiaas/publish?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=fiaas/publish&amp;utm_campaign=Badge_Coverage
+
+publish is a tool to package and release a python project. It will create a changelog and upload artifacts to Github and PyPI.
+
+It is created for and by the `Fiaas project`_, and used for most of our projects.
+
+.. _`Fiaas project`: https://github.com/fiaas
```

Comment:
Nitpick, but let's standardize on FIAAS in all caps?

Reply:
Fair enough, I'll fix.